src/components/Fact/Fact.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import {object} from 'prop-types';
-import {Text, View} from 'react-native';
+import {Text, View,} from 'react-native';
 
 import styles from './Fact.styles';
 
 const Fact = ({fact: {_id, text, upvotes}}) => (
-  <View key={_id} style={styles.container}>
-    <Text style={styles.text}>{text}</Text>
-    <View>
-      <Text>Upvotes</Text>
-      <Text>{upvotes}</Text>
+    <View key={_id} style={styles.container}>
+      <Text style={styles.text}>{text}</Text>
+      <View>
+        <Text>Upvotes</Text>
+        <Text>{upvotes}</Text>
+      </View>
```

Comment:
The only thing that changed here is the indentation right? We usually use 2 spaces as indentantion. You can configure that in your vscode